docs/installation.md

```diff
@@ -224,7 +224,7 @@ The source file `index.js` will be compiled (bundled) and a browser window opens
 
 #### Mithril bundler
 
-Mithril comes with a bundler tool of its own. It is sufficient for ES5-based projects that have no other dependencies other than Mithril, but it's currently considered experimental for projects that require other NPM dependencies. It produces smaller bundles than webpack, but you should not use it in production yet.
+Mithril comes with a bundler tool of its own. It is sufficient for ES5-based projects that have no other dependencies other than Mithril, but it's currently considered experimental for projects that require other NPM dependencies. It produces smaller bundles than webpack, but you should not use it in production yet--it is designed primarily as an internal build tool.
```

Comment:
This change is based on response to feedback I got when I commented on the build tool in a separate issue (#2136). Figured I may as well mention it while I was in there.